vyper/ast/nodes.py

```diff
@@ -112,6 +113,12 @@ def sortkey(key):
     )
 
 
+def _raise_syntax_exc(msg, ast_struct):
+    raise SyntaxException(
```

Comment:
Should check if fields are in `ast_struct` first, raising just `msg` if it's missing, else the full message